cmd/kubent/main_test.go

```diff
@@ -99,18 +99,26 @@ func TestStoreCollectorError(t *testing.T) {
 }
 
 func TestMainExitCodes(t *testing.T) {
+	expectedJsonOutput, _ := os.ReadFile(filepath.Join(FIXTURES_DIR, "expected-json-output.json"))
+	helm2FlagDisabled := "--helm2=false"
+	helm3FlagDisabled := "--helm3=false"
+	clusterFlagDisabled := "--cluster=false"
 	testCases := []struct {
 		name     string
 		args     []string // file list
 		expected int      // number of manifests
+		stdout   string   // stdout
 	}{
-		{"success", []string{"-c=false", "--helm2=false", "--helm3=false"}, 0},
-		{"errorBadFlag", []string{"-c=not-boolean"}, 2},
-		{"successFound", []string{"-c=false", "--helm2=false", "--helm3=false", "-f=" + filepath.Join(FIXTURES_DIR, "deployment-v1beta1.yaml")}, 0},
-		{"exitErrorFlagNone", []string{"-c=false", "--helm2=false", "--helm3=false", "-e"}, 0},
-		{"exitErrorFlagFound", []string{"-c=false", "--helm2=false", "--helm3=false", "-e", "-f=" + filepath.Join(FIXTURES_DIR, "deployment-v1beta1.yaml")}, 200},
-		{"version short flag set", []string{"-v"}, 0},
-		{"version long flag set", []string{"--version"}, 0},
+		{"success", []string{clusterFlagDisabled, helm2FlagDisabled, helm3FlagDisabled}, 0, ""},
+		{"errorBadFlag", []string{"-c=not-boolean"}, 2, ""},
+		{"successFound", []string{"-o=json", clusterFlagDisabled, helm2FlagDisabled, helm3FlagDisabled, "-f=" + filepath.Join(FIXTURES_DIR, "deployment-v1beta1.yaml")}, 0, string(expectedJsonOutput)},
+		{"exitErrorFlagNone", []string{clusterFlagDisabled, helm2FlagDisabled, helm3FlagDisabled, "-e"}, 0, ""},
+		{"exitErrorFlagFound", []string{clusterFlagDisabled, helm2FlagDisabled, helm3FlagDisabled, "-e", "-f=" + filepath.Join(FIXTURES_DIR, "deployment-v1beta1.yaml")}, 200, ""},
+		{"version short flag set", []string{"-v"}, 0, ""},
+		{"version long flag set", []string{"--version"}, 0, ""},
+		{"empty text output", []string{}, 0, ""},
+		{"empty json output", []string{"-o=json"}, 0, `[]
```

Comment:
Wouldn't `"[]\n"` work here? Assuming this is about the new line...

Reply:
yes can do